nc_time_axis/__init__.py

```diff
@@ -87,7 +92,7 @@ def __call__(self, x, pos=0):
 
 class NetCDFTimeDateLocator(mticker.Locator):
     """
-    Determines tick locations when plotting cftime.datetime data.
+    Determines tick locations when plotting cftime datetime data.
```

Comment:
Can this have consistent formatting with the [CalendarDateTime docstring](https://github.com/SciTools/nc-time-axis/pull/49/files#diff-250670b4e3f86a94858f6fdc62f01aceR34-R37)?